apps/blockchain/lib/blockchain/block.ex

```diff
@@ -13,21 +13,25 @@ defmodule Blockchain.Block do
   alias MerklePatriciaTree.DB
 
   # Defined in Eq.(18)
-  defstruct [
-    block_hash: nil,   # Hash for this block, acts simply as a cache.
-    header: %Header{}, # B_H
-    transactions: [],  # B_T
-    ommers: [],        # B_U
-  ]
+  defstruct block_hash: nil,
+
+            # Hash for this block, acts simply as a cache.
```

Comment:
@germsvel wrong comment

Reply:
Updated in 62a5ef4ebb7a6a5c6ea3c458b7f2ce51a170199d